test/offsetarray.jl

```diff
@@ -268,8 +268,7 @@ am = map(identity, a)
 # other functions
 v = OffsetArray(v0, (-3,))
 @test_approx_eq v v
-@test parent(v') == v0'
```

Comment:
seems worth keeping that in the spirit of what it used to be testing on offset arrays